### DISCUSSION TITLE
Do v4.3.0 and v4.3.1 work with .NET 8?

### DISCUSSION THREAD

**Discussion by ivebeenlinuxed** [ical-org/ical.net#640] [Q&A]
Steps to reproduce:

- Add the package via nuget
```
dotnet add package Ical.Net
```

- The command adds the following reference:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net8.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Ical.Net" Version="4.3.1" />
```

Error:

``` The type of namespace 'Ical' could not be found (are you missing a using directive or an assembly reference?)```

Workaround fix:

- Change the version to 4.2.0
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net8.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Ical.Net" Version="4.2.0" />
```

**Comment by axunonb** [comment]
> The type of namespace 'Ical' could not be found (are you missing a using directive or an assembly reference?)

This error is caused by user code.
ICal.Net definitely works with net8.0. See the sample app attached and
 give a try.
[TestAppNet8.zip](https://github.com/user-attachments/files/17690898/TestAppNet8.zip)
